### PR TITLE
chore: update angular 13

### PR DESCRIPTION
**Pull request by syl987**
### Requirements

Please read the contribution guidelines :notebook_with_decorative_cover: beforehand and if in doubt reach out to us :pray:

#### Definition of Done
**Mandatory for all PRs** 
- [ ] All unit and build tests are green 🚦 
- [ ] Reviewed and approved by maintainer :+1:

**Applicable to some PRs** 
- [ ] W3C AA accessibility fulfilled
- [ ] Documentation and examples are updated
- [ ] Tested on all supported browsers (see `Getting Started`)
- [ ] Unit tests with good coverage (e.g., >80%)
- [ ] No unintended visual changes. If needed, run the visual regression testing (see `README.md`)

In case there is any line item that you don't fullfil please add an explanation why. 

Any pull request that does not include enough information to be reviewed in a timely manner may be closed at the maintainers' discretion.

### CODE REVIEW

File: projects/ngx-docs-ui/src/lib/stack-blitz/stack-blitz-writer.ts [hunk 1]
```diff
@@ -70,14 +70,18 @@ const dependencies = {
...
-  'object-fit-images': '^3.2.4',
-  'rxjs': '~6.6.0',
-  'tslib': '^2.3.0',
-  'zone.js': '~0.10.2',
+  'rxjs': '^6.6.7',
+  'tslib': '^2.3.1',
+  'zone.js': '~0.11.4',
   'libphonenumber-js': '^1.9.38',
   'i18n-iso-countries': '^6.8.0'
 };
```
Comment (syl987) [inline projects/ngx-docs-ui/src/lib/stack-blitz/stack-blitz-writer.ts]: i have no idea what this is for.

File: projects/ng-aquila/src/number-stepper/number-stepper.component.scss [hunk 2]
```diff
@@ -98,8 +98,6 @@ $margin-default: nx-spacer(s);
 }
 
 .nx-stepper__inner-wrapper {
-  // IE11 height fix
-  height: nx-line-height(xs);
```
Comment (yd-allianz) [inline projects/ng-aquila/src/number-stepper/number-stepper.component.scss:102]: This seems like it's not necessarily IE-only, maybe only remove the comment here? Unless you checked that it's still OK without this height.

Comment (syl987) [reply projects/ng-aquila/src/number-stepper/number-stepper.component.scss]: reverted.

File: projects/ng-aquila/src/progress-stepper/multi/multi-step.component.ts [hunk 3]
```diff
@@ -55,6 +60,10 @@ export class NxMultiStepperComponent extends NxProgressStepperDirective
   }
   private _direction: NxMultiStepperDirection = 'horizontal';
 
+  constructor(_cdr: ChangeDetectorRef, _dir: Directionality, _el: ElementRef<HTMLElement>, @Inject(DOCUMENT) _document?: any) {
+    super(_cdr, _dir, _el, _document);
+  }
+
```
Comment (syl987) [inline projects/ng-aquila/src/progress-stepper/multi/multi-step.component.ts:66]: for some reason, some components did not have appropriate constructors.

File: projects/ng-aquila/tsconfig.build.json [hunk 4]
```diff
@@ -4,15 +4,13 @@
...
     "declarationMap": true,
-    "module": "es2015",
     "moduleResolution": "node",
     "declaration": true,
     "sourceMap": true,
     "inlineSources": true,
     "experimentalDecorators": true,
     "importHelpers": true,
     "types": [],
-    "lib": ["dom", "es2015"],
```
Comment (syl987) [inline projects/ng-aquila/tsconfig.build.json:15]: already specified in the parent file

**Pull request closed without merging**